frontend/src/metabase/visualizations/visualizations/Table.jsx

```diff
@@ -420,6 +421,7 @@ export default class Table extends Component {
           data={data}
           isPivoted={isPivoted}
           sort={sort}
+          limit={limit}
```

Comment:
Since `card` is already passed in `{...this.props}` maybe we could extract limit inside `TableSimple`.